### DISCUSSION TITLE
👍News: YouTube Updates Blocking to Instantly Remove User's Comments

### DISCUSSION THREAD

**Discussion by ThioJoe** [ThioJoe/YT-Spammer-Purge#753] [General]
According to the latest YouTube creators update video ([here](https://www.youtube.com/watch?v=on_3-M8uhFc)), now when you block a user from your channel, all their comments are instantly removed!

I'll have to test to ensure this works as described, but it should make the program's feature to "also remove other comments by user" obsolete.

**Comment by TechStudent10** [comment]
Finally, YouTube is fixing something that everyone was annoyed about.

**Reply by KendallDoesCoding** [reply]
Indeed!

**Comment by KendallDoesCoding** [comment]
Shouldn't this discussion category be "Announcements"?
